ilc/server/tailor/configs-injector.js

```diff
@@ -54,10 +54,11 @@ module.exports = class ConfigsInjector {
     #getRouteStyleRefsToPreload = async (reqUrl) => {
         const registryConf = await this.#registry.getConfig();
         const route = await this.#router.getRouteInfo(reqUrl);
+        const baseTemplate = await this.#registry.getTemplate(route.template);
```

Comment:
Is there is a chance that here we'll get template that is different (new version) from the one we used inside `router.getTemplateInfo`?